plugins/Login/Emails/PasswordResetEmail.php

```diff
@@ -49,20 +49,37 @@ private function setUpEmail()
 
         $this->setSubject($this->getDefaultSubject());
         $this->addReplyTo($replytoEmailAddress, $replytoEmailName);
-        $this->setWrappedHtmlBody($this->getDefaultBodyText());
+        $this->setWrappedHtmlBody($this->getHTMLBody());
+        $this->setBodyText($this->getDefaultBodyText());
     }
 
     private function getDefaultSubject()
     {
         return Piwik::translate('Login_MailTopicPasswordChange');
     }
 
-    private function getDefaultBodyText()
+    /**
+     * Get the translated plain text email body with the reset link
+     *
+     * @return string
+     */
+    private function getDefaultBodyText() : string
+    {
+        return Piwik::translate('Login_MailPasswordChangeBody2',
+            [Common::sanitizeInputValue($this->login), Common::sanitizeInputValue($this->ip), Common::sanitizeInputValue($this->resetUrl)]);
+    }
+
+    /**
+     * Create the HTML email body from the plain text body
+     *
+     * @return string
+     */
+    private function getHTMLBody() : string
     {
         return '<p>' . str_replace(
             "\n\n",
-            "</p><p>",
-            Piwik::translate('Login_MailPasswordChangeBody2', [Common::sanitizeInputValue($this->login), Common::sanitizeInputValue($this->ip), Common::sanitizeInputValue($this->resetUrl)])
+            "</p><p style='word-break: break-all'>",
```

Comment:
any reason for not putting a `<a>` tag around the link? 🤔

Reply:
:+1: That would be a better user experience, I've committed a change to add the tag.